neural_network/simple_neural_network.py

```diff
@@ -0,0 +1,53 @@
+"""
+Forward propagation explanation:
+https://towardsdatascience.com/forward-propagation-in-neural-networks-simplified-math-and-code-version-bbcfef6f9250
+"""
+
+import random as ran
+import math
+
+
+# Sigmoid
+def sigmoid_function(value: float, deriv: bool = False) -> float:
+    """Return the sigmoid function of a float.
+
+    >>> sigmoid_function(3.5)
+    0.9706877692486436
+    >>> sigmoid_function(3.5, True)
+    -8.75
+    """
+    if deriv:
+        return value * (1 - value)
+    return 1 / (1 + math.exp(-value))
+
+
+# Initial Value
+INITIAL_VALUE = 0.02
+
+# Output
+expected = int(input("What is the expected value? (1-100): "))
+
+expected = expected / 100
+
+# Number of propagations
+n = int(input("How many propagations?: "))
+
+# Random weight
+syn0 = 2 * (ran.randint(1, 34)) - 1
```

Comment:
What does this name mean?

Reply:
I changed the name of this variable to be more descriptive.